packages/flutter/lib/src/widgets/tap_region.dart

```diff
@@ -33,12 +33,24 @@ bool _tapRegionDebug(String message, [Iterable<String>? details]) {
   return true;
 }
 
-/// The type of callback that [TapRegion.onTapOutside] and
-/// [TapRegion.onTapInside] take.
-///
-/// The event is the pointer event that caused the callback to be called.
+/// Signature for a callback called for a [PointerDownEvent] relative to a [TapRegion].
+/// 
```

Comment:
Oh I think the analyzer check is failing because here and below there is a trailing space at the end of the line.

Reply:
Thanks! Fixed that now